### ISSUE TITLE
Rotating device resets camera

### ISSUE THREAD

**Issue by gviddy** [amay077/Xamarin.Forms.GoogleMaps#413]
I have noticed that if I pan and zoom into a position and then I rotate the device then the camera resets the view to the default startup position and zoom. Not the current position and zoom.

I am guessing that Forms does some kind of new Layout when the device is rotated so everything gets reset.

How do you all get around this ?

**Comment by BradleyAlcock** [comment]
I'm experiencing similar behavior.

@amay077 as requested, I've attached screenshots on this issue.
![img1](https://user-images.githubusercontent.com/27420409/33009214-d48e2b7c-cdde-11e7-9753-6edd84c02d1e.jpg)

first screen, shows the map as first loaded.

![img2](https://user-images.githubusercontent.com/27420409/33009241-edf96c8e-cdde-11e7-839c-ccc07fdae0c3.jpg)

second screen shows the moved view

![img3](https://user-images.githubusercontent.com/27420409/33009271-022692fe-cddf-11e7-8abf-a287f55884b7.jpg)

third screen returns the view. Ignore the icon color change, I found that reloading the data was causing the icons to change.

![img4](https://user-images.githubusercontent.com/27420409/33009309-20ef7dd6-cddf-11e7-890d-9f369416ea1b.jpg)


changing back to the first height doesn't fix the issue.

the arrow button is used to change the bounds. first tap:
AbsoluteLayout.SetLayoutBounds(MyMap, new Rectangle(0, 1, 1, 0.908));
this sets the height/position to take up the majority of the screen.
Second tap:
AbsoluteLayout.SetLayoutBounds(MyMap, new Rectangle(0, 0.15, 1, 0.39));
changes the height/position back.
